sanity/schema/singleTypes/global.tsx

```diff
@@ -0,0 +1,148 @@
+import {defineField, defineType} from 'sanity'
```

Comment:
[GLOBALLY] Change the prettier configuration to make:
`import {defineField, defineType} from 'sanity'` into `import { defineField, defineType } from 'sanity'`
for more readability.